packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -408,6 +434,10 @@ function _getWeb3AndUrlForBot(botConfig) {
   }
 }
 
+async function _getBlockNumberOnChainIdMultiChain(botConfig, chainId) {
+  return await new Web3(botConfig?.environmentVariables[`NODE_URL_${chainId}`]).eth.getBlockNumber();
+}
```

Comment:
nit: any reason not to use the retry config logic as done in _getWeb3AndUrlForBot (we can always do this in a follow up)?

Reply:
`_getWeb3AndUrlForBot` expects a format of provider that we dont use in across. this is janky but is done to avoid needing to modify the bot configs to have redundant information. we ONLY support `NODE_URL_n` in the relayer right now, which `_getWeb3AndUrlForBot` does not work with.